modules/registry-third-party-registries.adoc

```diff
@@ -5,18 +5,34 @@
 [id="registry-third-party-registries_{context}"]
 = Third-party registries
 
-{product-title} can create containers using images from third-party registries,
-but it is unlikely that these registries offer the same image notification
-support as the integrated {product-title} registry. In this situation,
-{product-title} will fetch tags from the remote registry upon imagestream
-creation.
+{product-title} can create containers using images from third-party registries, but it is unlikely that these registries offer the same image notification support as the integrated {product-title} registry. In this situation, {product-title} will fetch tags from the remote registry upon imagestream creation.
 
-Refreshing the fetched tags is as simple as running `oc import-image
-<stream>`. When new images are detected, the previously described build and
-deployment reactions occur.
+Refreshing the fetched tags is as simple as running `oc import-image <stream>`. When new images are detected, the previously described build and deployment reactions occur.
```

Comment:
What's simple for you may be hard for me. Try to avoid value judgement words like "simple" and "easy." It can lead to frustration in the reader.

Suggestion:
To refresh the fetched tags, run `oc import-image <stream>`.

Reply:
Acked